src/integrations/authentication/definitions/types.ts

```diff
@@ -7,13 +7,11 @@ export type Identity = {
     email_verified: boolean;
 };
 
-export type Token = string;
```

Comment:
For the database implementation we've chosen to keep using this 'useless' types to make it more clear (and changeable in to future).